### PR TITLE
Finish all Modals + Styling

### PR DESCRIPTION
**Pull request by dhkaneda**
Quite a few things done since the last PR:

- Skeletonned out rest of modals
- Filled in all models with inputs or hardcoded placeholders
- Styled all models save for tour scheduling
- Implemented scheduling selection few ways before deciding on final
- Finished styling and cleaned up 

TODO: refactor to CSS Modules or Styled Components - vanilla css file would be difficult to maintain and is not locally scoped - will cause collisions/conflicts in future. 

@phuctranlv @dontpressthat @richard-c-gill please take a look. Just wish to merge for sake of progress/backup. Refactoring to be done.

### CODE REVIEW

File: client/src/assets/arrow.svg [hunk 1]
```diff
@@ -3,6 +3,6 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 width="100px" height="100px" viewBox="0 0 100 100" enable-background="new 0 0 100 100" xml:space="preserve">
-<path fill="none" stroke="#000000" stroke-width="6" stroke-miterlimit="10" d="M61.432,36.095V19.396l28.824,27.633L61.432,74.662
+<path fill="none" stroke="#0055CC" stroke-width="7" stroke-miterlimit="10" d="M61.432,36.095V19.396l28.824,27.633L61.432,74.662
```
Comment (phuctranlv) [inline client/src/assets/arrow.svg:6]: I'm super impressed! You actually work on your svg edits and creation...

File: client/src/components/Agent.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <input type="radio" id={agent.id} />
+        <label htmlFor={agent.id}>
+          <span id="avatar"><img src={img} alt="avatar"></img></span>
+        </label>
+        <label htmlFor={agent.id}>
+          <div className="agent-title">{agent.title}</div>
+          <div className="agent-name">{agent.name}</div>
+          <div>
+            <img src={star} alt="star" height="10" />
+            {`${agent.rating}/5 Rating`}
```
Comment (phuctranlv) [inline client/src/components/Agent.jsx:29]: Thanks for this! I totally did not know we can have JS inside the return statement here...

File: client/src/components/Summary.jsx [hunk 3]
```diff
@@ -4,89 +4,104 @@
...
-        </div>
-        <div className="footer">
-          <button type="button" onClick={this.showModal}>contact agent</button>
-          <button type="button" onClick={this.showModal}>take a tour</button>
+          <div className="footer">
+            <button type="button" onClick={() => this.showModal('contact')}>Contact agent</button>
+            <button type="button" onClick={() => this.showModal('tour')}>Take a tour</button>
+          </div>
         </div>
         {modal}
```
Comment (phuctranlv) [inline client/src/components/Summary.jsx:106]: nice! learned something new here

File: client/src/utils/getDates.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable arrow-body-style */
+const daysController = (days, delta) => {
+  let alteredDays;
+
+  if (!delta) {
+    const today = new Date();
+    const tomorrow = new Date(new Date().getTime() + 24 * 60 * 60 * 1000);
```
Comment (phuctranlv) [inline client/src/utils/getDates.js:7]: I'm curious if there's a better way to do this?

**Review by phuctranlv — APPROVED** [review]
Looks great overall!

File: client/src/utils/getDates.js (hunk 4 above)
Comment (dhkaneda) [reply client/src/utils/getDates.js:7]: I wondered about this, but I left it for the sake of clarity. 
If you look at the day controller, you'll see I used a different way to change the date but that stays in seconds format. And that's the state I keep for modifying. The abbreviated way is needed for rendering.

File: client/src/assets/arrow.svg (hunk 1 above)
Comment (dhkaneda) [reply client/src/assets/arrow.svg:6]: Oh, actually I did all of this in Adobe Illustrator. If you save your vectors as SVG, it's really just all code. 
AHEM I MEAN yes I coded it all out

File: client/src/components/Summary.jsx (hunk 3 above)
Comment (dhkaneda) [reply client/src/components/Summary.jsx:106]: I wonder if there's a better way to do this, but it did work.

**Pull request merged**